### ISSUE TITLE
Question: How to disable a mock namespace?

### ISSUE THREAD

**Issue by bjjblackbelt** [cpputest/cpputest#1540]
Hello,

I am using CPPUTest v4.0. In my tests, I mock two production code functions defined similarly to the following:

```cpp
int time_now()
{
    mock("time").actualCall("time_now").returnIntValueOrDefault(0);
}

int spi_write(spi_t* spi_ptr)
{
    mock("spi").actualCall("spi_write")
               .withParameterOfType("spi_t", "spi_ptr", spi_ptr)
               .returnIntValueOrDefault(0);
}
```
The production code function I am testing is called `cycle()` which calls both of these mocked functions. However, I only care about verifying the behavior of the `spi_write()` function. In one test I am attempting the following, but I am getting a mock failure for unexpected calls to `time_now()`.

```cpp
TEST(test_spi_behavior, test1)
{
    mock("spi").expectOneCall("spi_write").withParameterOfType("spi_t", "spi_ptr", &spi_handle).andReturnValue(0);
    mock("time").disable();
    cycle();
}
```
Failures are also seen when using `mock("time").ignoreOtherCalls()`. I would like to be able to disable failing a test because of the 'unexpected calls' to `time_now()`. Is it possible to achieve this, but also have the mocked call to `time_now()` return the default value?

**Comment by basvodde** [comment]
Shouldn't the spi_write have a return statement ?

**Comment by bjjblackbelt** [comment]
Yes, that is a mistake in the example above for both the functions. However, is it possible to disable a specific mock nameplate?

**Comment by basvodde** [comment]
Namespace? Yes, you can.
